modules/configuring-shared-cpus-for-a-workload.adoc

```diff
@@ -0,0 +1,106 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-shared-cpus-for-a-workload_{context}"]
+= Configuring shared CPUs in a performance profile
+
+You can define and enable shared CPUs by using a performance profile. You can use these shared CPUs for lightweight application tasks.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+//* You enabled the `TechPreviewNoUpgrade` feature set on the cluster. For more information about //enabling a feature set, see the _Additional resources_ section.
+// +
+// [WARNING]
+// ====
+// Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
+// ====
+
+.Procedure
+
+. Create a `PerformanceProfile` resource:
+
+.. Create a YAML file that defines the `PerformanceProfile` resource:
++
+.Example `shared-cpu-pp.yaml` file
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: example-shared-cpu-pp
+spec:
+  cpu:
+    reserved: "0-1"
+    isolated: "4-8"
+    shared: "2-3" <1>
+  workloadHints:
+    mixedCpus: true <2>
+  nodeSelector:
+    node-role.kubernetes.io/performance: "test" <3>
+----
+<1> Define the shared CPU cores.
+<2> Enable the shared CPUs feature by enabling the `mixedCpus` workload hint.
+<3> Select the node that you want to use for shared CPUs.
+
+.. Create the `PerformanceProfile` resource by running the following command:
++
+[source,bash]
+----
+$ oc create -f shared-cpu-pp.yaml
+----
++
+.Example output
+[source,terminal]
+----
+performanceprofile.performance.openshift.io/example-shared-cpu-pp created
+----
+
+
+.Verification
+
+. Verify that the performance profile is created by running the following command:
```

Comment:
Let's also mention the user that this requires node reboot, so we need to wait for mcp to get updated and nodes to get ready.

Reply:
Done